recipes/xorg-xproto/bld.bat

```diff
@@ -0,0 +1,7 @@
+:: Just delegate to the Unixy script; but some of the X.org tarballs have
+:: config.guess files too ancient to recognize 64-bit Windows!
+curl -sSL -o config.guess "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess"
+curl -sSL -o config.sub "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub"
```

Comment:
Alternatively since this seems to be necessary for other Xorg packages. Maybe it is worthwhile to package these bits in some other package (e.g. `xorg-build-helpers` or something). That way they can be tracked in one package, but used elsewhere.